src/main/java/com/minecolonies/coremod/colony/managers/StatisticAchievementManager.java

```diff
@@ -259,7 +273,7 @@ public void checkAchievements()
     }
     
     @Override
-    public void triggerAchievement(@NotNull final MineColoniesAchievement achievement, @NotNull final Colony colony)
+    public void triggerAchievement(@NotNull final MineColoniesAchievement achievement)
     {
         /*if (this.colonyAchievements.contains(achievement))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)